roxctl/deployment/check/check.go

```diff
@@ -162,8 +165,16 @@ func (d *deploymentCheckCommand) Construct(_ []string, cmd *cobra.Command, f *pr
 }
 
 func (d *deploymentCheckCommand) Validate() error {
-	if _, err := os.Open(d.file); err != nil {
-		return common.ErrInvalidCommandOption.CausedBy(err)
+	if d.file != "" {
+		if _, err := os.Open(d.file); err != nil {
+			return common.ErrInvalidCommandOption.CausedBy(err)
+		}
+	} else {
+		for _, file := range d.files {
+			if _, err := os.Open(file); err != nil {
```

Comment:
You probably want to use `errorhelpers.ErrorList` to check all files before returning the error

Reply:
Good to know, I used it now.